src/python/build_management/source_mapper.py

```diff
@@ -225,6 +225,9 @@ def normalize_source_path(path):
   # out/<build_dir>/ at time of build generation (chromium only).
   path = utils.remove_prefix(path, '../../')
 
+  # Remove /proc/self/cwd prefix if exists.
```

Comment:
Can you mention that this is something bazel can add?

Reply:
Done!